### PR TITLE
Confirm Saving and Save Playing Track

### PR DESCRIPTION
**Pull request by binaryoverload**
- [x] Completed PR - Has all features or fixes that it was meant to and isn't half done
- [x] Follows the style guide - Style guide found [here](https://github.com/FlareBot/FlareBot/blob/master/CONTRIBUTING.md)
- [x] All changes have been tested

## Added
 - Confirmation on overwriting of an existing playlist
 - Seek Command to move the place in a song
 - Info and success helper methods
 - Vevo specific error message for restricted videos

## Changed
 - Refactored `getUsage` method to be `sendUsage` due to the result not being needed
 - Refactored the whole codebase to use the `sendXMessage` helper methods
 - Youtube errors now log with a `NO_ANNOUNCE` flag

## Deprecated
N/A

## Removed
N/A

## Fixed
 - Current song now is saved with the playlist
 - Fix error catching

## Security
N/A

### CODE REVIEW

File: src/main/java/stream/flarebot/flarebot/commands/music/SeekCommand.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    }
+
+    @Override
+    public String getCommand() {
+        return "seek";
+    }
+
+    @Override
+    public String getDescription() {
+        return null;
```
Comment (weeryan17) [inline src/main/java/stream/flarebot/flarebot/commands/music/SeekCommand.java]: That shoudn't be null. provide an actual description.

File: src/main/java/stream/flarebot/flarebot/util/MessageUtils.java [hunk 2]
```diff
@@ -153,6 +152,16 @@ public static Message sendWarningMessage(String message, MessageChannel channel)
                 .complete();
     }
 
+    public static Message sendSuccessMessage(String message, TextChannel channel) {
```
Comment (weeryan17) [inline src/main/java/stream/flarebot/flarebot/util/MessageUtils.java]: Why have these? I see no point.

File: src/main/java/stream/flarebot/flarebot/util/MessageUtils.java [hunk 3]
```diff
@@ -153,6 +152,16 @@ public static Message sendWarningMessage(String message, MessageChannel channel)
...
+        return channel.sendMessage(MessageUtils.getEmbed().setColor(Color.YELLOW).setDescription(message).build())
+                .complete();
+    }
+
+    public static Message sendInfoMessage(String message, TextChannel channel) {
```
Comment (weeryan17) [inline src/main/java/stream/flarebot/flarebot/util/MessageUtils.java]: /\

File: src/main/java/stream/flarebot/flarebot/commands/music/SeekCommand.java (hunk 1 above)
Comment (binaryoverload) [reply src/main/java/stream/flarebot/flarebot/commands/music/SeekCommand.java]: Duhhhh

File: src/main/java/stream/flarebot/flarebot/util/MessageUtils.java (hunk 2 above)
Comment (binaryoverload) [reply src/main/java/stream/flarebot/flarebot/util/MessageUtils.java]: We send a lot of success and info messages and we always write the same code, I plan to move over to helper messages a lot more

**Pull request merged**